hooks/playbooks/manila_create_default_resources.yml

```diff
@@ -9,6 +9,5 @@
         PATH: "{{ cifmw_path }}"
       ansible.builtin.shell: |
         oc -n {{ namespace }} exec -it pod/openstackclient \
-          -- openstack share type create default false
-        oc -n {{ namespace }} exec -it pod/openstackclient \
-          -- openstack share type set default --extra-specs snapshot_support=True
+          -- openstack share type create default false \
+             --snapshot-support True --create-share-from-snapshot-support True
```

Comment:
i guess you can keep the same level of indentation of line 12.

Reply:
Done; thanks!